src/covid_shared/cli_tools.py

```diff
@@ -237,7 +237,7 @@ def _wrapped(*args, **kwargs):
 
 def update_with_previous_metadata(run_metadata: RunMetadata, input_root: Path) -> RunMetadata:
     """Convenience function for updating metadata from an input source."""
-    key = str(input_root).replace(' ', '_').replace('-', '_').lower() + 'metadata'
+    key = str(input_root.resolve()).replace(' ', '_').replace('-', '_').lower() + '_metadata'
```

Comment:
looks like a backwards incompatible change. doesn't bother me in the least though